realtime_ai_character/audio/speech_to_text/whisper.py

```diff
@@ -25,7 +25,6 @@ def __init__(self, use='local'):
         super().__init__()
         if use == 'local':
             logger.info(f"Loading [Local Whisper] model: [{config.model}]...")
-            self.model = whisper.load_model(config.model)
```

Comment:
good find, this is unnecessary.  Can you change the log above to "Using [Local Whisper] ..." as well? Just so it's more clear.

Reply:
for a new model though, preloading still helps. But the self.model is indeed confusing. I will update your PR and merge.